data/aliases.ts

```diff
@@ -264,9 +265,25 @@ export const Aliases: {[alias: string]: string} = {
 	oricoriopsychic: "Oricorio-Pa'u",
 	lycanrocmidday: "Lycanroc",
 	lycanrocday: "Lycanroc",
+	lycanm: "Lycanroc-Midnight",
 	lycanrocn: "Lycanroc-Midnight",
 	lycanrocnight: "Lycanroc-Midnight",
+	lycand: "Lycanroc-Dusk",
 	lycanrocd: "Lycanroc-Dusk",
+	muka: "Muk-Alola",
+	alomuk: "Muk-Alola",
+	markwaka: "Marowak-Alola",
+	alowak: "Marowak-Alola",
+	sandslasha: "Sandslash-Alola",
+	snowslash: "Sandslash-Alola",
+	ninetalesa: "Ninetales-Alola",
+	alotales: "Ninetales-Alola",
+	raticatea: "Raticate-Alola",
+	golema: "Golem-Alola",
+	alolem: "Golem-Alola",
+	neckboy: "Exeggutor-Alola",
+	persiana: "Persian-Alola",
```

Comment:
i think this is a redundant addition (alongside with muka, sandslasha, etc), since there's code in the species#get function that accounts for alola/hisui/galar/paldea formes already

Reply:
gotcha i'll fix that